packages/react-core/src/components/Alert/Alert.tsx

```diff
@@ -48,9 +51,18 @@ export const Alert: React.FunctionComponent<AlertProps & OUIAProps> = ({
   title,
   children = '',
   className = '',
+  truncateTitle,
   ouiaId,
   ...props
 }: AlertProps & OUIAProps) => {
+  const titleRef = React.useRef(null)
+  React.useEffect(() => {
+    if(!Boolean(titleRef.current) || !Boolean(truncateTitle)) {
+      return ;
+    }
+    titleRef.current.style.setProperty(maxLines.name, truncateTitle.toString())
+  }, [titleRef, truncateTitle])
+
```

Comment:
Curious why you chose to use a `titleRef` Vs applying the modifier directly to `className`?

Reply:
@dlabrecq the use of `titleRef` is to inject a new value to the `*-title--max-lines` variable which indicates the number of lines to show in the title.

I could not find any way to set it as `className`. If you know a way to do so, let me know. :smiley_cat: